src/main/java/com/cloudbees/maven/license/ProcessMojo.java

```diff
@@ -146,11 +149,20 @@ public void execute() throws MojoExecutionException {
             }
         }
 
-        if (generateLicenseXml!=null)
-            comp.add((LicenseScript) shell.parse(getClass().getResourceAsStream("xmlgen.groovy"),"xmlgen.groovy"));
+        if (generateLicenseXml!=null) {
+            try {
+                comp.add((LicenseScript) shell.parse(getClass().getResource("xmlgen.groovy").toURI()));
```

Comment:
Is using `getResource` the fix here?

Reply:
the method parameters changed, InputStream isn't valid anymore, I chose the URI  method